testdata/default/cheats/Prompt.t.sol

```diff
@@ -15,4 +16,14 @@ contract PromptTest is DSTest {
         vm._expectCheatcodeRevert();
         vm.promptSecret("test");
     }
+
+    function testPrompt_Address() public {
+        address test = vm.promptAddress("test");
+        console.log(test);
+    }
+
+    function testPrompt_Uint() public {
+        uint256 test = vm.promptUint("test");
+        console.log(test);
+    }
```

Comment:
let's improve these tests, and assert that what gets parsed is actually what we expect

Reply:
added assertEq with what I am prompting as an input.